### ISSUE TITLE
SchemaPrinter generates default values for mutations

### ISSUE THREAD

**Issue by DixiOmnes** [graphql-dotnet/graphql-dotnet#2194]
Hello

We use SchemaPrinter in unit tests to validate schema integrity.

In queries and graphtypes we have a lot of mandatory scalar types which SchemaPrinter displays without default values like this:

> type Client {
>   Id: Long!
>   addresses: [Address]!
>   birthDay: Date!
> }

but for mutations it generates default values like this:

> parametersCorrection(applicationId: Guid! = 00000000-0000-0000-0000-000000000000, date: Date! = 01/01/0001 00:00:00, paymentDay: Int! = 0): Application!

We found this when unit test on local machine and build server generated different default date values due to different locales - _Date! = 01/01/0001_ and _Date! = 01.01.0001_
Unfortunately, we were not able to find any options to control this behavior and it looks like bug.

**Comment by sungam3r** [comment]
@Shane32 Please look at this bug. It can be fixed as part of #2321.
```csharp
   public string FormatDefaultValue(object value, IGraphType graphType)
        {
            return graphType switch
            {
                NonNullGraphType nonNull => FormatDefaultValue(value, nonNull.ResolvedType),
                ListGraphType list => "[{0}]".ToFormat(string.Join(", ", ((IEnumerable<object>)value).Select(i => FormatDefaultValue(i, list.ResolvedType)))),
                IInputObjectGraphType input => FormatInputObjectValue(value, input),
                EnumerationGraphType enumeration => (enumeration.Serialize(value) ?? throw new ArgumentOutOfRangeException(nameof(value), $"Unknown value '{value}' for enumeration '{enumeration.Name}'")).ToString(),
                ScalarGraphType _ => value switch  <--- change to AstPrinter.Print(scalarType.ToAST())
                {
                    string s => $"\"{s}\"",
                    bool b => b ? "true" : "false",
                    _ => value.ToString() // TODO: how to print custom scalars ("") ?
                },
                _ => throw new NotSupportedException($"Unsupported graph type '{graphType}'")
            };
        }
```

Also see ` private string FormatInputObjectValue(object value, IInputObjectGraphType input)`. It can help you to write default implementation for `InputObjectGraphType.ToAST`

**Comment by sungam3r** [comment]
@DixiOmnes Fixed and will be published in v4 soon.
